src/services/tracker-state.js

```diff
@@ -119,6 +119,28 @@ export default class TrackerState {
     return _.get(this.itemsForLocations, [generalLocation, detailedLocation]);
   }
 
+  getLocationsForItem(itemName) {
+    const generalLocations = _.keys(this.itemsForLocations).filter(
+      (general) => _.keys(this.itemsForLocations[general]).find(
+        (detailed) => this.itemsForLocations[general][detailed] === itemName,
```

Comment:
To be more consistent with existing code, we should do `_.get(this.itemsForLocations, [generalLocation, detailedLocation])`.